Tasks/Inc/RTOS_BPS.h

```diff
@@ -103,6 +103,15 @@ void RTOS_BPS_DelaySecs(uint16_t dly);
  */
 void RTOS_BPS_DelayMs(uint16_t dly);
 
+/**
+ * @brief Creates a semaphore with the initially specified count
+ * 
+ * @param sem - pointer to a semaphore object to create and initialize
+ * @param name - name of the semaphore
+ * @param count - initial count for the semaphore
+ */
+void RTOS_BPS_SemCreate(BPS_OS_SEM* sem, char* name, uint32_t count);
```

Comment:
FreeRTOS offers the option to add a max count that the sempahore can reach (before throwing an error or something). I would add something to the documentation saying it would be useful to add a certain constant in case we are improperly using a sempahore (when we do switch over). This happened with our CAN semaphore where it went up to the 100's because we were never posting it.